satori/datasources/datasource_data_access_controller.go

```diff
@@ -0,0 +1,79 @@
+package datasources
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/satoricyber/terraform-provider-satori/satori/api"
+)
+
+func DatasourceDataAccessController() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceDataAccessControllerRead,
+		Description: "Find DAC by type, region, cloud provider and unique name.",
+		Schema: map[string]*schema.Schema{
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "DAC's type. The available values are: PRIVATE, PRIVATE_MANAGED or PUBLIC.",
+			},
+			"region": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "DAC's region.",
+			},
+			"cloud_provider": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "DAC's cloud provider.",
+			},
+			"unique_name": {
```

Comment:
After second thought, let's remove it at all.

You can query the DAC with:
1. ID (for private)
2. type region provider (for public only)

Make sense?

Reply:
We can also make a validation for input